### DISCUSSION TITLE
Qiitaイベントの曜日間違い

### DISCUSSION THREAD

**Discussion by takizawa0548** [increments/qiita-discussions#887] [要望・アイデア・不具合]
!![Screenshot_20240722-070416~2.png](https://github.com/user-attachments/assets/69aff63e-c150-4a06-8a84-36addf18d089)

トップに表示される画像がFRIDAYになってます。

**Comment by tanaka-asami415** [comment]
@takizawa0548 
ご報告ありがとうございます！修正いたします🙇

**Reply by kabechiyo** [reply]
@takizawa0548 
こちら修正が完了いたしました！
ご報告いただきありがとうございました🙏
